.github/workflows/rundoc.yml

```diff
@@ -12,30 +12,52 @@ jobs:
       contents: write
```

Comment:
Now that we're using the Linguist GitHub App for auth, we can remove the `permissions` block here, since increasing the permissions granted to the default `GITHUB_TOKEN` is no longer required.